recipes/tclreadline/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "tclreadline" %}
+{% set version = "2.3.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flightaware/tclreadline/archive/v{{ version }}.tar.gz
+  sha256: a64e0faed5957b8e1ac16f179948e21cdd6d3b8313590b7ab049a3192ab864fb
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

Have you tried building for macOS? It normally just works
```suggestion
  skip: true  # [win]
```

---

Yep, I tried but it failed, and I have no idea why, here's the log: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=104261&view=logs&j=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&t=81967b5f-0283-5c0b-87c1-e8fdeef842a3

---

Looks like it's finding the headers from conda-forge but linking to the library from the macOS SDK when running autotools. Let's see if this fixes it.

---

Looks like it's finding the headers from conda-forge but linking to the library from the macOS SDK when running autotools. Let's see if this fixes it.